docs/rules/arrow-spacing.md

```diff
@@ -12,12 +12,12 @@ This rule normalize style of spacing before/after of arrow functions arrow(`=>`)
 
 ## Rule Details
 
-this rules takes one arguments of structure contains `before` and `after` property
+this rule takes one arguments of structure contains `before` and `after` property
 and each property has bool value.
 
 default configuration is `{ "before": true, "after": true }`.
 
-`true` means there should have **one space** and `false` means **no space**.
+`true` means there should have **spaces** and `false` means **no space**.
```

Comment:
I had to read this a few times to understand, perhaps a slight rewording might help:

```
`true` means there should be **one or more spaces** and `false` means **no spaces**.
```